<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TX717-4441WGD" name="Default test" tests="1" failures="1" timestamp="13 Apr 2017 14:38:33 GMT" time="6.668" errors="0">
  <testcase name="runTC_SF_OBOpportunitySharing" time="6.59" classname="testscripts.SF.SmokeTestScripts.TC_SF_OBOpportunitySharing">
    <failure type="com.cognizant.framework.FrameworkException" message="The test case &amp;quot;TC_SF_OBOpportunitySharing&amp;quot; is not found in the Business Flow sheet!">
      <![CDATA[com.cognizant.framework.FrameworkException: The test case "TC_SF_OBOpportunitySharing" is not found in the Business Flow sheet!
	at com.cognizant.Craft.DriverScript.initializeBusinessFlow(DriverScript.java:551)
	at com.cognizant.Craft.DriverScript.initializeTestScript(DriverScript.java:536)
	at com.cognizant.Craft.DriverScript.executeCraft(DriverScript.java:161)
	at com.cognizant.Craft.DriverScript.driveTestExecution(DriverScript.java:152)
	at testscripts.SF.SmokeTestScripts.TC_SF_OBOpportunitySharing.runTC_SF_OBOpportunitySharing(TC_SF_OBOpportunitySharing.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- runTC_SF_OBOpportunitySharing -->
</testsuite> <!-- Default test -->
